.github/workflows/release.yaml

```diff
@@ -2,7 +2,6 @@ name: release
 
 on:
   workflow_dispatch:
-    branches: [ main ]
```

Comment:
@esloch I've added this trying to fix the release, but I think it's not required at all